R/opts.R

```diff
@@ -533,6 +534,7 @@ gp_opts <- function(basis_prop = 0.2,
                     ls_sd = 7,
                     ls_min = 0,
                     ls_max = 60,
+                    ls = Normal(mean = 0.5, sd = 0.1, max = 1),
```

Comment:
Noting that from a scientific view point I don't think this is a very good prior or very easy for people to reason on.